bundles/EcommerceFrameworkBundle/FilterService/FilterType/ElasticSearch/MultiSelect.php

```diff
@@ -56,14 +56,20 @@ public function addCondition(AbstractFilterDefinitionType $filterDefinition, IPr
                     $quotedValues[] = $v;
                 }
             }
-            if (!empty($quotedValues)) {
```

Comment:
the not empty check got lost now? is that not necessary?

Reply:
ah... sorry - its required for the "else" block - readded the empty check